os/mm/mm_heap/mm_partition_mgr.c

```diff
@@ -83,13 +83,15 @@ void mm_initialize_ram_partitions(void)
 	uint32_t start;
 	uint32_t size;
 
-	start = (uint32_t)__ksram_segment_start__ + (uint32_t)__ksram_segment_size__;
-	size = (REGION_SIZE - (uint32_t)__ksram_segment_size__) / CONFIG_NUM_APPS;
+	start = (uint32_t)__usram_segment_start__ + (uint32_t)__usram_segment_size__;
+	size = (REGION_END - start) / CONFIG_NUM_APPS;
```

Comment:
This comment is about below ALIGN_EVEN..

1. I dont think it align's an odd address. 
    example, addr: 0x1001 .. result is also 0x1001 only..
2.  Even alignment is not sufficient..  It requires 4Bytes up alignment.. 

3. FYI, mask values for alignment.. 
   2 Byte(even) : 1
    4 Bytes(word) : 3
    8 Bytes(stack alignment on ARM) : 7

Reply:
Done.